.github/workflows/test.yml

```diff
@@ -2,6 +2,9 @@ name: home-assistant.io Test
 
 on: [push, pull_request]
 
+permissions:
```

Comment:
This should be set in repo settings, not here.

Reply:
Took care of it. Thanks